src/mscorlib/src/System/Runtime/InteropServices/GcHandle.cs

```diff
@@ -40,8 +46,13 @@ public enum GCHandleType
     [StructLayout(LayoutKind.Sequential)]
     public struct GCHandle
     {
-        // IMPORTANT: This must be kept in sync with the GCHandleType enum.
-        private const GCHandleType MaxHandleType = GCHandleType.Pinned;
+#if BIT64
+    private const long NoFlag = ~1L;
```

Comment:
Can this be just `~(nint)1` without ifdef ? Also, since it is used in just one place, it may not be worth a constant.